packages/api/src/auth/decoders/ethereum.ts

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken'
+
+export const ethereum = (token: string) => {
+  if (process.env.NODE_ENV === 'production') {
+    if (!process.env.JWT_SECRET) {
+      throw new Error('`JWT_SECRET` env var is not set.')
+    }
+
+    try {
+      const secret = process.env.JWT_SECRET as string
```

Comment:
Could you rename the env var to ETHEREUM_JWT_SECRET to align with the supabase implementation? (But wait until #1652 is resolved)

Reply:
Done. Updated here and in the readme as well

Also, I updated the example during this round as well. Here is the new link https://redwood-ethereum-login-demo.vercel.app/ Had to publish this to my personal verdaccio to getting a working demo deployed. After this is released I plan to update versions to the public published packages